### PR TITLE
MODULE-2

### PR DESCRIPTION
**Pull request by reviakin**

### CODE REVIEW

File: src/task2/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const bookCSVFilePath = resolve(process.cwd(), "src/task2/csv/book.csv");
+const bookTXTFilePath = resolve(process.cwd(), "src/task2/txt/book.txt");
+
+const readStream = createReadStream(bookCSVFilePath);
+readStream.on("error", console.error);
+
+const writeStream = createWriteStream(bookTXTFilePath);
+writeStream.on("error", console.error);
+
+readStream.pipe(csvtojson()).pipe(writeStream);
```
Comment (Kuznietsov) [inline src/task2/index.ts]: Could you, please, look at the format of the input and the output at the example carefully. They have some differences and you need to implement the code to produce such an output.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+node_modules
+dist
+src/task2/txt/**
```
Comment (Kuznietsov) [inline .gitignore]: Could you, please, add the rule for excluding `csv` files too.

**Review by Kuznietsov — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (reviakin) [reply .gitignore]: @Kunzetsov the file was updated

File: src/task2/index.ts (hunk 1 above)
Comment (reviakin) [reply src/task2/index.ts]: @Kunzetsov, Should content from the "txt" file have a valid JSON structure?

Comment (reviakin) [reply src/task2/index.ts]: Added PreFileLine callback to make a table's headers to lower case

**Review by Kuznietsov — APPROVED** [review]
Code LGTM. Good job 👍

**Pull request merged**